lib/generators/hyrax/templates/db/migrate/20200617002029_change_sipity_entity_specific_responsibility.rb.erb

```diff
@@ -0,0 +1,11 @@
+class ChangeSipityEntitySpecificResponsibility < ActiveRecord::Migration<%= migration_version %>
```

Comment:
i'd really like to know whether it's possible for existing applications (ones that are already using postgres successfully) to get by without running this migration. i.e. is this migration optional for existing apps? or is this a hard break?

Reply:
I suspect that this is something that _should_ be run on those applications.  Without the change, I believe some of the Sipity queries may fail.  Given that this is "in production" it is also possible that people have remediated their applications already. It is certainly a "HEY - This is something to consider"

Triage steps will be:

1) Look at the `sipity_entity_specific_responsibilities` table and see if there are records.  If there are no records, run this without fear.
2) If there are records, you'll want to back up your data before you run it.

My suspicion is that no one has entries in this table, but I'm not certain. And yes, we could remove the table, but then we lose a key function of sipity (which is not fully integrated with Hyrax and is something we need to look at in regards to the general permission ecosystem).

Reply:
As I've been debugging, this table is most definitely used in `app/services/hyrax/workflow/workflow_factory.rb` with the `assign_specific_roles_to`.